src/plugins/data_source/server/cryptography/cryptography_client.test.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { CryptographyClient } from './cryptography_client';
+import { randomBytes } from 'crypto';
+
+const dummyWrappingKeyName = 'dummy_wrapping_key_name';
+const dummyWrappingKeyNamespace = 'dummy_wrapping_key_namespace';
+
+test('Invalid wrapping key size throws error', () => {
```

Comment:
It's being blocked by some strange issue when Jest invoke the encrypt function, so I've created an [issue](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2068) to track it and am following up with the sde owner. Since this PR is blocking pen test, I would prefer a fast follow up.

Reply:
added